ports/mimxrt/Makefile

```diff
@@ -106,8 +122,9 @@ SRC_TINYUSB_C += \
 	lib/tinyusb/src/portable/chipidea/ci_hs/dcd_ci_hs.c \
 	lib/tinyusb/src/tusb.c
 
-# All settings for Ethernet support are controller by the value of MICROPY_PY_LWIP
+# All settings for Ethernet support are controller by the value of MICROPY_PY_LWIP and the supported MCUs
 ifeq ($(MICROPY_PY_LWIP),1)
+ifeq ($(MCU_SERIES),$(filter $(MCU_SERIES), MIMXRT1021 MIMXRT1052 MIMXRT1062 MIMXRT1064 MIMXRT1176))
```

Comment:
I don't know about this, prefer to control it via `MICROPY_PY_LWIP`, sometimes you need to disable it for a board.

Reply:
That's an old setting related to Ethernet. It might result to the same, since all these boards have LWIP enabled, and the MIMXRT1011 and MIMXRT1015 do not use LWIP.